function-controller/pkg/controller/deployer.go

```diff
@@ -93,7 +93,11 @@ func (d *deployer) buildMainContainer(function *v1.Function) corev1.Container {
 
 func (d *deployer) buildSidecarContainer(function *v1.Function) corev1.Container {
 	c := corev1.Container{Name: "sidecar"}
-	c.Image = sidecarImage + ":" + os.Getenv("RIFF_FUNCTION_SIDECAR_TAG")
+	imageName := os.Getenv("RIFF_FUNCTION_SIDECAR_IMAGE")
+	if imageName == "" {
+		imageName = sidecarImage
+	}
+	c.Image = imageName + ":" + os.Getenv("RIFF_FUNCTION_SIDECAR_TAG")
```

Comment:
should this default to "latest" if the env var is empty/missing?

Reply:
I'm not a big fan of hardwiring "latest" with it's magic behavior in k8s into the code.
Do you know if there is a way to default to the same tag as the running (function-controller) image?
Since we build all the core images together it makes sense to default to the same version.

Reply:
We do that for the Helm chart, so let's just leave as is for now